site/diagrams/sequenceDiagrams/tenant_getTenant_Success.txt

```diff
@@ -0,0 +1,9 @@
+title Get Tenant Information (success)
+
+Client->+Tenant Service: transfer("tenant", "get", tenant-id)
+
+note over Tenant Service:
+lookup tenant
+end note
+
+Tenant Service->-Client: transfer("registration/${reply-to}", "200", tenantMetaData)
```

Comment:
You updated the wrong address "registration" to "tenant" already, but not the image.
So, in the web pages this is still displayed wrong, please provide the updated image as well.